src/main/java/org/bytedeco/javacpp/tools/Generator.java

```diff
@@ -3028,7 +3028,7 @@ void parametersAfter(MethodInformation methodInfo) {
             if ("void*".equals(typeName[0]) && !methodInfo.parameterTypes[j].isAnnotationPresent(Opaque.class)) {
                 typeName[0] = "char*";
             }
-            
+
```

Comment:
Please revert unnecessary changes in formatting here and there

Reply:
Although this looks like it's removing white space, so that's good thing. No need to revert

Reply:
Yes.